router/core/router.go

```diff
@@ -508,7 +508,9 @@ func (r *Router) listenAndServe(cfg *nodev1.RouterConfig) error {
 }
 
 func (r *Router) initModules(ctx context.Context) error {
-	for _, moduleInfo := range modules {
+	var moduleList = SortModules(modules)
+
+	for _, moduleInfo := range moduleList {
```

Comment:
Is this the only place where we access the modules?

Reply:
No, we access it in RegisterModule, but should not be a problem 
```
if _, ok := modules[string(mod.ID)]; ok {
    panic(fmt.Sprintf("module already registered: %s", mod.ID))
}
modules[string(mod.ID)] = mod
```